tests/integration/deploying/deploy_command_test.py

```diff
@@ -28,5 +30,18 @@ async def test_deploying_contract(
     response = await protostar.deploy(
         contract=compiled_contract_filepath, gateway_url=devnet_gateway_url, inputs=[42]
     )
+    assert response.address is not None
 
+    response = await protostar.deploy(
```

Comment:
Could you please move this logic to separate test case?

Reply:
[address feedback in tests](https://github.com/software-mansion/protostar/pull/1140/commits/dd3248b2c6e8d02b1338c84e78c81b39e1ca0225)